doc/installing.md

```diff
@@ -165,6 +165,13 @@ sudo apt update
 sudo apt install acl attr autoconf automake dnsmasq-base git golang libacl1-dev libcap-dev liblxc1 liblxc-dev libsqlite3-dev libtool libudev-dev liblz4-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables
 ```
 
+```{note}
+If you are using the `liblxc-dev`` package and having compile time errors during `go-lxc` module building, please
+ensure that `LXC_DEVEL` value is `0` for your `liblxc` build. To check that, look at `/usr/include/lxc/version.h`.
+If `LXC_DEVEL` value is `1` you can replace it with `0` to workaround the problem. It's a packaging bug and
+we are aware of it for Ubuntu Jammy/Kinetic. Ubuntu Lunar/Mantic are fine.
```

Comment:
```suggestion
If you use the `liblxc-dev` package and get compile time errors when building the `go-lxc` module, 
ensure that the value for `LXC_DEVEL` is `0` for your `liblxc` build. To check that, look at `/usr/include/lxc/version.h`.
If the `LXC_DEVEL` value is `1`, replace it with `0` to work around the problem. It's a packaging bug, and
we are aware of it for Ubuntu 22.04/22.10. Ubuntu 23.04/23.10 does not have this problem.
```

Reply:
Seems the diff doesn't highlight it, but the source for the spellcheck failure is the second backtick ` after `liblxc-dev`.

Reply:
Huge thanks, Ruth! I've updated the PR.